internal/install/static_snapshot_yml.go

```diff
@@ -69,4 +69,31 @@ services:
     depends_on:
       package-registry:
         condition: service_healthy
+
+  elastic-agent:
+    image: docker.elastic.co/beats/elastic-agent:${STACK_VERSION}
+    depends_on:
+      elasticsearch:
+        condition: service_healthy
+      kibana:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD", "grep", "Agent is starting", "/usr/share/elastic-agent/elastic-agent.log"]
+      retries: 30
+      interval: 1s
+    environment:
+    - "FLEET_ENROLL=1"
+    - "FLEET_ENROLL_INSECURE=1"
+    - "FLEET_SETUP=1"
+    - "KIBANA_HOST=http://kibana:5601"
+    volumes:
+    - type: bind
+      source: ../tmp/
+      target: /tmp/service_logs/
```

Comment:
Ok, so with this solution, we lose the opportunity to test on real filesystem and have log files distributed across the entire partition. We might need a comment somewhere about this limitation or a task for next iteration (maybe we can solve it somehow).

Reply:
Not necessarily. There is no requirement for log files paths in the dataset configuration to be restricted to `/tmp/service_logs`. So, for example, when we write a test for the `system` package, we can refer to standard locations for system log files assuming the Agent is running some flavor of Linux.